includes/class-wc-payments-checkout.php

```diff
@@ -347,9 +347,14 @@ public function get_enabled_payment_method_config() {
 	 * @return bool - True if the payment method is reusable and the saved cards feature is enabled for the gateway and there is no subscription item in the cart, false otherwise.
 	 */
 	private function should_upe_payment_method_show_save_option( $payment_method ) {
+		if ( $payment_method->get_id() === Payment_Method::CARD && is_user_logged_in() && WC_Payments_Features::is_woopay_enabled() ) {
```

Comment:
What about non UPE merchants? Did all merchants get migrated to UPE or WooPay is not available for non UPE merchants?

Reply:
The `showSaveOption` [here](https://github.com/Automattic/woocommerce-payments/blob/914cef65fd3e35a47ed9e5cc2004790bef9621a8/includes/class-wc-payments-checkout.php#L324) is the one that shows the save card option and it references only this method, also I couldn't downgrade to non-UPE and didn't find any related things on the WCPay DevTools, I found the following P2: paJDYF-9xf-p2

@pierorocca could you confirm that we only use UPE on WCPay right now?